### PR TITLE
Add new voice_id field to Problem model.

### PR DESCRIPTION
**Pull request by madyasiwi**
New lab voice over feature adds VoiceId attribute into quiz-block element. This element is stored and retrieved from the Problem model, and as it is currently it has no field to accommodate storing this information.

This is a fix for article #2 of the following issue:
https://app.asana.com/0/64071763853597/67761473071006

### CODE REVIEW

**Comment by polesye** [comment]
@madyasiwi where this field is used?
I'm not sure that we need this field in a new system. Could you check it?

**Comment by polesye** [comment]
@auraz FYI

**Comment by madyasiwi** [comment]
@polesye, https://labster.slack.com/files/indra/F0FQS5P27/quizblocks_tel_tissueengineering.xml
It is the VoiceId attribute from Tissue Engineering Lab quiz block. This is the first of its kind and my impression is we need this to be enabled on our new edX also, CMIIW.

Edit: Just checked with Franky, and he confirmed that the feature will also be used on the new edX.

Edit2: I currently have no idea yet about where or how the data is stored/processed in the new edX though.

**Comment by polesye** [comment]
@madyasiwi It looks so.

File: common/djangoapps/labster/masters.py [hunk 1]
```diff
@@ -203,6 +204,8 @@ def get_problem_as_platform_xml(problem):
     }
 
     extra = {}
+    if problem.voice_id:
+        extra['VoiceId'] = str(problem.voice_id)
```
Comment (auraz) [inline common/djangoapps/labster/masters.py:208]: why not unicode instead of str?

**Comment by madyasiwi** [comment]
I saw the other fields are treated the same, and it has not broken as far as I can tell.

**Comment by polesye** [comment]
Unity takes these XML files directly from s3. I'm not sure if it is really neccessary to have this field in a new Labter API. @auraz what do you think?

**Comment by madyasiwi** [comment]
@polesye If it is the case then the voice over feature should just work with the new edX, and we don't need to change anything about it.

**Comment by polesye** [comment]
@madyasiwi yes, but we have to verify that. Maybe someove from contaent team can help you? `@franky` , for example

File: common/djangoapps/labster/masters.py (hunk 1 above)
Comment (madyasiwi) [inline common/djangoapps/labster/masters.py:208]: Just to be consistent to the surrounding context. More over, the data is an attribute value and is not meant to -- or should never -- be a translatable content.

Comment (auraz) [inline common/djangoapps/labster/masters.py:208]: OK, thanks

**Comment by polesye** [comment]
@madyasiwi You have 2 (from me and @auraz) :+1: so you can go ahead and merge.

**Comment by madyasiwi** [comment]
@polesye the content team will be more than happy to help in any way the could. But it is not readily apparent to me how they can help us verify the thing will run on the new edX server in this context. But if it helps, they are testing it locally when developing this feature and the quizblock xml file is read locally, unaltered.

**Comment by polesye** [comment]
@madyasiwi this PR can me merged.

**Comment by madyasiwi** [comment]
@polesye Thanks, will proceed after I verify some stuff on the staging server.

**Pull request merged**